### PR TITLE
Optimize the size of report

### LINKED ISSUE #964: Optimize the size of report
We need the report to have a size that is possible to send via email. I.e. under 5mb even in case there is many matchings and around 50 pairs. At the moment the size of report is > 30  Mb 

Look into how to decrese this (it might mean losing some nice graphics) but mostly there might be loads of suboptimal pdf generation there)

### PR DESCRIPTION
**Pull request by krllstdn**
Closes #964

### CODE REVIEW

File: txmatching/database/services/report_service.py [hunk 1]
```diff
@@ -175,6 +175,7 @@ def generate_pdf_report(
             '--margin-left': '0',
             '--margin-right': '0',
             '--margin-bottom': '0',
+            '--dpi': ' 60'
```
Comment (kubantjan) [inline txmatching/database/services/report_service.py:178]: please attach an example report, so we can see that the dpi is not too low for print

**Review by kubantjan — CHANGES_REQUESTED** [review]
and add test

File: txmatching/database/services/report_service.py (hunk 1 above)
Comment (krllstdn) [reply txmatching/database/services/report_service.py:178]: [report_2022_09_07_07_31_05.pdf](https://github.com/mild-blue/txmatching/files/9504213/report_2022_09_07_07_31_05.pdf)

**Comment by krllstdn** [comment]
with lowered dpi : [report_2022_09_07_07_31_05.pdf](https://github.com/mild-blue/txmatching/files/9504193/report_2022_09_07_07_31_05.pdf)
with original dpi: [report_2022_09_07_10_30_30.pdf](https://github.com/mild-blue/txmatching/files/9504229/report_2022_09_07_10_30_30.pdf)

Personally, I don't see any difference in how they look, but the one with the original dpi is 25% bigger

**Comment by kubantjan** [comment]
> @kubantjan with lowered dpi : [report_2022_09_07_07_31_05.pdf](https://github.com/mild-blue/txmatching/files/9504193/report_2022_09_07_07_31_05.pdf) with original dpi: [report_2022_09_07_10_30_30.pdf](https://github.com/mild-blue/txmatching/files/9504229/report_2022_09_07_10_30_30.pdf)
> 
> Personally, I don't see any difference in how they look, but the one with the original dpi is 25% bigger

@krllstdn  have you tried then dpi 30? or even dpi 1? maybe the setting does not matter and will only decrese the size of the report then?

**Comment by krllstdn** [comment]
@kubantjan Yes I did. Dpi 1 ruined everything, dpi 30 some elements were too pixelated, dpi 60 looked optimal to me. But now I rechecked everything and I decided to stop on dpi 300. Because it doesn't increase size of the file significantly, while it's optimal for printing

**Review by kubantjan — APPROVED** [review]

**Review by kubantjan — CHANGES_REQUESTED** [review]
dopsat vysvetleni proc se tam dela ten magic

**Review by kubantjan — APPROVED** [review]

**Pull request merged**